coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -246,26 +246,36 @@ def test_batchnorm_3d(self, num_features, eps, affine, dynamic_input, backend):
             )
 
     @pytest.mark.parametrize(
-        "rank, num_features, eps, training, backend",
-        itertools.product([3, 4, 5], [5, 1], [0.1, 1e-05], [True, False], backends),
+        "rank, num_features, eps, running_vars_exist, training, backend",
+        itertools.product([3, 4, 5], [5, 1], [0.1, 1e-05], [True, False], [True, False], backends),
     )
-    def test_batchnorm_dynamic(self, rank, num_features, eps, training, backend):
+    def test_batchnorm_dynamic(self, rank, num_features, eps, running_vars_exist, training, backend):
         if backend != "neuralnetwork" and rank == 5 and num_features == 5:
             pytest.xfail("rdar://75770475 ([ActivateMIL] Failure in "
                          "test_ops_public_torch.py::TestBatchNorm::test_batchnorm_3d "
                          "[elementwise_kernel_cpu: Cannot broadcast])")
 
-        model = ModuleWrapper(
-            nn.functional.batch_norm,
-            {"training": training, "eps": eps,},
-        )
         input_shape = [6, num_features, 3, 4, 5]
         input_shape = input_shape[:rank]
         _input = torch.randn(*input_shape)
-        _mean = torch.randn(num_features)
-        _var = torch.randn(num_features)
 
-        inputs = (_input, _mean, _var)
+        if training == True and running_vars_exist == True:
```

Comment:
Nit: no `==` needed for boolean comparison.

Reply:
resolved.